src/index.js

```diff
@@ -1,2 +0,0 @@
-import _ from 'lodash';
-import './style.css';
```

Comment:
- [ ] Please use ES6 modules to write modular JavaScript with proper ES6 syntax.

* usage of arrow functions instead of the `function` keyword
* usage of `import` and `export` instead of `require` or `module.exports`
* proper usage of `const` and `let` keywords